tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# This is the cmake build file for the tests directory of the
+# NCEPLIBS-g2c_UTILS project.
+#
+# Ed Hartnett, 7/11/21
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -r8 -convert big_endian -assume byterecl")
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-0 -fdefault-real-8 -fconvert=big-endian")
+endif()
+
+include_directories(${PROJECT_SOURCE_DIR})
+
+add_executable(tst_g2_create tst_g2_create.c)
+add_test(NAME tst_g2_create COMMAND tst_g2_create)
```

Comment:
Use unique names for executable and test targets.

Reply:
I think it's fine to have the test name the same as the executable

Reply:
It is fine for now. But as soon as one starts to use same executable for different tests, this can be confusing.
This is just something I have learnt from experience.

Reply:
How can we use the same executable for different tests? Each test gets it's own executable...

Reply:
In this case yes.  There are instances when one uses a test harness, it is done differently.
Forget about the comment.